examples/base_and_running_difference_mapcubes.py

```diff
@@ -0,0 +1,61 @@
+"""
+==================================================
+Base Difference and Running Difference in Mapcubes
+==================================================
+
+This example illustrates how to do base and running differencing with a MapCube.
+Base differencing uses a fixed reference point when compared to running difference.
+"""
+
+import matplotlib.pyplot as plt
+import matplotlib.colors as colors
+import matplotlib.cm as cm
+
+import sunpy.map
+import sunpy.physics.differential_rotation as diffrot
+from sunpy.data.sample import AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE, AIA_193_CUTOUT03_IMAGE
+
+###############################################################################
+# We create the MapCube using the AIA_193_CUTOUT sample data.
+# To create a MapCube, we can call Map directly but add in a keyword to output a MapCube instead.
+aiamapcube = sunpy.map.Map(AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE,
+                           AIA_193_CUTOUT03_IMAGE, cube=True)
+
+############################################################################
+# We first loop through all the maps in the aiamapcube and differentially rotate each map
+# in the mapcube to the time of the base map in case of base difference.
+# We then subtract each map data from the base map and store all such difference maps in a list.
+# In case of running difference, we loop through all the maps in the aiamapcube and differentially rotate each map
+# in the mapcube with respect to the previous map.
+# We then subtract each map data from the previous map and store all such difference maps in a list.
+base_diffmap = []
+running_diffmap = []
+for i, map_i in enumerate(aiamapcube[1:]):
+    # take the difference
+    aiamap_rot = diffrot.diffrot_map(map_i, time=aiamapcube[0].date)
+    aiamapcube_rot = diffrot.diffrot_map(aiamapcube[i+1], time=aiamapcube[i].date)
+    diffdata = map_i.data - aiamap_rot.data
+    smap_base = sunpy.map.Map(diffdata, map_i.meta)
+    diffdata = aiamapcube_rot.data - map_i.data
+    smap_run = sunpy.map.Map(diffdata, map_i.meta)
+    smap_base.plot_settings['cmap'] = cm.get_cmap('Greys_r')
+    smap_base.plot_settings['norm'] = colors.LogNorm(100, smap.max())
+    smap_run.plot_settings['cmap'] = cm.get_cmap('Greys_r')
+    smap_run.plot_settings['norm'] = colors.LogNorm(100, smap.max())
```

Comment:
What does the output look like if we remove these plotting settings?

Reply:
It will not be normalized

Reply:
Sure, but does it make the output look better or worse?

Reply:
I think the differences are equally visible even if we don't normalize the plots.

Normalization came into picture , I guess when we are discussing about PT outputs

Reply:
Yeah, if we can remove it and not affect the outcome. I would prefer that. It makes it a bit more tidy. 

Can you run the examples without and with?

Reply:
sure, I will let you know in a minute

Reply:
![screenshot 68](https://user-images.githubusercontent.com/16035442/38028697-5110739e-32b1-11e8-9b31-ba5dbf7e8233.png)
![screenshot 69](https://user-images.githubusercontent.com/16035442/38028716-5a95b3a2-32b1-11e8-8ceb-01b0443eae49.png)


Without normalization

![screenshot 70](https://user-images.githubusercontent.com/16035442/38028789-94054f30-32b1-11e8-8607-96ab92def581.png)
![screenshot 71](https://user-images.githubusercontent.com/16035442/38028794-96f363f8-32b1-11e8-8710-12f30ae88212.png)

I think we may need normalization, maybe

Reply:
Yeah looks like that needs to stay.

Reply:
I think there should be a comment explaining what and why they have been added.